src/workerd/api/container.c++

```diff
@@ -48,24 +48,38 @@ void Container::start(jsg::Lock& js, jsg::Optional<StartupOptions> maybeOptions)
   }
 
   IoContext::current().addTask(req.sendIgnoringResult());
+  IoContext::current().addTask(rpcClient->monitorRequest(capnp::MessageSize{4, 0})
```

Comment:
We also need to automatically set up monitoring in the constructor if the container is already running.